assets/js/util/tracking/createEnableTracking.js

```diff
@@ -36,6 +41,21 @@ export default function createEnableTracking( config, dataLayerTarget ) {
 		document.head.appendChild( scriptTag );
 
 		dataLayerPush( 'js', new Date() );
-		dataLayerPush( 'config', config.trackingID );
+		dataLayerPush( 'config', config.trackingID, {
+			send_page_view: config.isSiteKitScreen,
+		} );
+		hasInsertedTag = true;
+	};
+
+	/**
+	 * Enables tracking.
+	 */
+	function enableTracking() {
+		initializeSnippet();
+	}
+
+	return {
```

Comment:
See my last comment regarding repurposing this module. I think we can keep it to returning the single function as before, just `initializeSnippet` instead. `createEnableTracking` should then be renamed of course to `createInitializeSnippet` and some tests would need to move around but I think this would be fairly straightforward to do.

Reply:
https://github.com/google/site-kit-wp/pull/4142/commits/4faffa2ddd8f92d4cf9aeae34ec7c9d2c948ce3f